core/src/main/java/org/bitcoinj/core/Wallet.java

```diff
@@ -1815,6 +1838,22 @@ private void receive(Transaction tx, StoredBlock block, BlockChain.NewBlockType
                 // this method has been called by BlockChain for all relevant transactions. Otherwise we'd double
                 // count.
                 ignoreNextNewBlock.add(txHash);
+
+                // When a tx is received from the best chain, if other txns that spend this tx are IN_CONFLICT,
```

Comment:
Maybe this chunk can be factored into a method? IntelliJ can do this sort of refactoring automatically.